source/WorkingPractices/kgo.rst

```diff
@@ -36,6 +21,62 @@ to this area is restricted to members of the Simulation Systems and Deployment
 Team and they will update the KGO files to include your new answers as part of
 the commit process.
 
+
+KGO Update Process
+------------------
+
+Getting the process right for KGO changing tickets significantly helps get
+such changes onto the trunk. When preparing your change for review:
```

Comment:
Worth saying (in the right place) that if you suspect a KGO change, run all as a precautionary measure?

Reply:
I've added something in the "preparing for review" section - may or may not be the right space, but its a start